filters/all.js

```diff
@@ -1,3 +1,7 @@
+const React = require('react');
```

Comment:
I see you're using Nunjucks filters. I'd avoid using them because we're gonna get rid of Nunjucks sooner than later. Maybe you should place all the logic in a `generate:after` hook and convert this template into a React template?

Reply:
You can see in commits that I had a template written in React, but due to this https://github.com/asyncapi/generator/issues/521 I wanna stay with Nunjucks. When mentioned issue will be fixed, rewritten to React will be very easy.

Reply:
then why not fixing https://github.com/asyncapi/generator/issues/521 first?

Reply:
I don't want to end the cycle without merging neither the component nor the html-template like in previous cycle 😄 Taks with bundler took a lot longer than I thought and the current PR would have opened in the past week. I want to deal with this task, but after done the current tasks.